### PR TITLE
MSP-2788: add accounting to consts and values

### PR DESCRIPTION
**Pull request by Uburro**

### CODE REVIEW

File: api/v1/slurmcluster_types.go [hunk 1]
```diff
@@ -295,6 +300,70 @@ type SlurmNodes struct {
...
+	// +kubebuilder:default=3306
+	Port int32 `json:"port"`
+	// Key defines the key of username and password in the secret
+	//
+	// +kubebuilder:validation:Optional
+	User string `json:"user"`
+	// Secret for connection string to the SlurmDBD database
+	//
+	// +kubebuilder:validation:Optional
+	Secret SecretName `json:"secret"`
```
Comment (asteny) [inline api/v1/slurmcluster_types.go]: Maybe just Secret?

File: api/v1/slurmcluster_types.go [hunk 2]
```diff
@@ -295,6 +300,70 @@ type SlurmNodes struct {
...
+}
+
+// SecretName defines the name of the secret
```
Comment (asteny) [inline api/v1/slurmcluster_types.go]: It not defines only name, so Secret?

File: api/v1/slurmcluster_types.go [hunk 3]
```diff
@@ -329,7 +398,6 @@ type SlurmNodeControllerVolumes struct {
 }
 
 // SlurmNodeWorker defines the configuration for the Slurm worker node
-// +kubebuilder:validation:XValidation:rule="self.slurmd.resources.memory > self.volumes.sharedMemorySize",message="resources memory of slurmd must be bigger than sharedMemorySize"
```
Comment (asteny) [inline api/v1/slurmcluster_types.go:332]: Why u delete this validation?

File: internal/naming/naming.go [hunk 4]
```diff
@@ -80,6 +80,14 @@ func BuildStatefulSetName(componentType consts.ComponentType, clusterName string
 	}.String()
 }
 
+func BuildDeploymentName(componentType consts.ComponentType, clusterName string) string {
```
Comment (asteny) [inline internal/naming/naming.go:83]: U do not use clusterName here

File: internal/consts/accounting.go [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package consts
+
+const (
+	SecretSlurmdbdConfigs       = "slurmdbd-configs"
+	HostnameAccounting          = "accounting"
+	DbdPort                     = "6819"
```
Comment (asteny) [inline internal/consts/accounting.go]: Why we have 2 vars with port and using only one?

File: internal/consts/annotation.go [hunk 6]
```diff
@@ -4,5 +4,6 @@ const (
 	AnnotationVersions = "versions"
 
 	AnnotationApparmorKey             = "container.apparmor.security.beta.kubernetes.io"
+	DefaultContainerAnnotationName    = "kubectl.kubernetes.io/default-container"
```
Comment (asteny) [inline internal/consts/annotation.go:7]: U want use this annotation in all our containers?

Comment (Uburro) [reply internal/consts/annotation.go:7]: Yes, right now default container is munge. It's inconvenient to specify a special container every time.

File: api/v1/slurmcluster_types.go (hunk 3 above)
Comment (Uburro) [reply api/v1/slurmcluster_types.go:332]: Cel validate int, not resources of memory, this leads to an error

File: internal/consts/annotation.go (hunk 6 above)
Comment (asteny) [reply internal/consts/annotation.go:7]: Does that mean that next PR will be about changing annotations for all containers?

Comment (Uburro) [reply internal/consts/annotation.go:7]: Not next, but after accounting merged - yes

**Pull request merged**